src/views/Vote/Vote.pug

```diff
@@ -9,18 +9,115 @@
 
   DWBH is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
-  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the[Vue warn]: Error in mounted hook (Promise/async): "SyntaxError: Failed to execute 'querySelector' on 'Document': '[object HTMLCollection]' is not a valid selector."
+
   GNU General Public License for more details.
 
   You should have received a copy of the GNU General Public License
   along with DWBH.  If not, see <https://www.gnu.org/licenses/>.
 
-.vote
-  h2.title(data-testid="title") {{ $t("VIEWS.VOTE.TITLE") }}
-
-  dw-vote-form(
-    v-bind:group="group"
-    v-bind:voting="voting"
-    v-bind:initScore="initScore"
-    data-testid="vote-form"
+.vote(
+  v-bind:class="getVoteClasses(voteScore)"
+)
+  .voting-interaction-mobile(
+    v-touch:swipe="swipeHandler"
+    v-bind:class="{ voted: voteSelected, 'animation-true': animationTrue, 'landscape': landscape, 'loaded': isLoaded }"
   )
+    .voting-title
+      .close-x
+        include ../../assets/img/cross.svg
+      .voting-smile
+        dw-smiley(
+          :voteScore="voteScore"
+        )
+      .swipe-action
+        .text-upper
+          span.comment-share {{ $t("VIEWS.VOTE.SHARE") }}
+
+        .text(
+          v-on:click="toggleMoodSelected()"
+        )
+          include ../../assets/img/up.svg
+          span.comment-share {{ $t("VIEWS.VOTE.COMMENT_SHARE") }}
+          span.select-mood {{ $t("VIEWS.VOTE.SELECT_MOOD") }}
+
+        .arrow(
+          v-on:click="worsenMood(1)"
```

Comment:
1 seems unnecessary, you mean increase or decrease a variable by 1. Why don't use `variable++` or `variable--`